### PR TITLE
Write thesis objective section.

### PR DESCRIPTION
**Pull request by daffidwilde**
Addresses the first point in #132 about the introduction needing more backbone. In particular, we (the examiners and I) liked my explanation "utilisation" being split into three parts, each addressed by a chapter of the thesis.

### CODE REVIEW

File: chapters/intro/main.tex [hunk 1]
```diff
@@ -35,33 +35,66 @@ \chapter{Introduction}
...
+disease (COPD). COPD is a respiratory condition with known links to deprivation,
+and often presents as a comorbidity, i.e. in concurrence with at least one other
+condition. These affiliations make living with and treating COPD inherently
+difficult, emphasising the importance of studying it closely.
+
+This thesis considers how clustering can be utilised to better understand the
+COPD population treated by \ctmuhb. In order to carry out this utilisation, this
+thesis incorporates three themes from modern OR: algorithm evaluation,
+clustering and segmentation, and operational healthcare modelling. These
+seemingly disparate themes are presented as a triptych, where each theme
```
Comment (drvinceknight) [inline chapters/intro/main.tex:82]: I had to google `triptych`, it's [as I expected -- you're good with them there words] apparently fine but perhaps `triplet` ?

File: chapters/intro/main.tex [hunk 2]
```diff
@@ -35,33 +35,66 @@ \chapter{Introduction}
...
+corresponds to its own chapter. With each themed chapter, there is a clear
+contribution to a component of utilising machine learning.
+
+The broader issue of algorithm evaluation is addressed Chapter~\ref{chp:edo},
```
Comment (drvinceknight) [inline chapters/intro/main.tex:86]:
```suggestion
The broader issue of algorithm evaluation is addressed in Chapter~\ref{chp:edo},
```

File: chapters/intro/main.tex [hunk 3]
```diff
@@ -35,33 +35,66 @@ \chapter{Introduction}
...
+\ctmuhb\ are particularly concerned with understanding the operational
+characteristics of their patients presenting chronic obstructive pulmonary
+disease (COPD). COPD is a respiratory condition with known links to deprivation,
+and often presents as a comorbidity, i.e. in concurrence with at least one other
+condition. These affiliations make living with and treating COPD inherently
+difficult, emphasising the importance of studying it closely.
+
+This thesis considers how clustering can be utilised to better understand the
+COPD population treated by \ctmuhb. In order to carry out this utilisation, this
+thesis incorporates three themes from modern OR: algorithm evaluation,
```
Comment (drvinceknight) [inline chapters/intro/main.tex:80]:
```suggestion
COPD population treated by \ctmuhb. In order to carry this out, this
thesis incorporates three themes from modern OR: algorithm evaluation,
```

File: chapters/intro/main.tex [hunk 4]
```diff
@@ -35,33 +35,66 @@ \chapter{Introduction}
...
+clustering and segmentation, and operational healthcare modelling. These
+seemingly disparate themes are presented as a triptych, where each theme
+corresponds to its own chapter. With each themed chapter, there is a clear
+contribution to a component of utilising machine learning.
+
+The broader issue of algorithm evaluation is addressed Chapter~\ref{chp:edo},
+where a novel framework for understanding the objective quality of an algorithm
+is presented. In Chapter~\ref{chp:kmodes}, an extension to an existing
+clustering algorithm is devised and subsequently evaluated using the framework
```
Comment (drvinceknight) [inline chapters/intro/main.tex:89]: It took me a little while to put my finger on what I think is niggling me in this paragraph.

You say in the previous paragraph:

>  With each themed chapter, there is a clear contribution to a component of utilising machine learning.
 
So I was expecting to see the next three chapters be directly corresponding to creation, evaluation and application.

In reality `chp:edo` and `chp:kmodes` do both "creation" and "evaluation" don't they? 

This of course isn't a problem, just the narrative doesn't flow as well.

**Review by drvinceknight — CHANGES_REQUESTED** [review]

File: chapters/intro/main.tex (hunk 4 above)
Comment (daffidwilde) [reply chapters/intro/main.tex]: Fixed by 12904ef

**Pull request merged**